src/gwt/src/org/rstudio/studio/client/rsconnect/ui/RSConnectAccountList.java

```diff
@@ -41,14 +41,16 @@ public class RSConnectAccountList extends Composite implements CanSetControlId
    public RSConnectAccountList(RSConnectServerOperations server, 
          GlobalDisplay display,
          boolean refreshImmediately,
-         boolean showCloudAccounts,
```

Comment:
This was a really confusingly named variable, because it actually doesn't refer to Posit Cloud. I should have renamed it in October, but I was lazy